Client/Frontend/Browser/SearchLoader.swift

```diff
@@ -50,7 +50,7 @@ class _SearchLoader<UnusedA, UnusedB>: Loader<Cursor<Site>, SearchViewController
             let deferred = self.profile.history.getSitesByFrecencyWithHistoryLimit(100, bookmarksLimit: 5, whereURLContains: query)
             inProgress = deferred as? Cancellable
```

Comment:
This cast never succeeds. Which breaks my patch of course.

Reply:
Remember me mentioning that cancellation support was removed and not replaced? Yeah, that.